scripts/project_udfs_to_step_udfs.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+import sys
+from EPPs.common import StepEPP
+
+
+class ProjectUDFStepUDF(StepEPP):
```

Comment:
Does this script support only single project? 
If yes it should be added to the _validate_step

Reply:
updated